### ISSUE TITLE
Building a new DateTime changes the value of DateTime.MinValue

### ISSUE THREAD

**Issue by jbatte47** [nbuilder/nbuilder#17]
The following code produces interesting results:

``` csharp
var generatedDateTime = Builder<DateTime>.CreateNew().Build();
var minDateTime = DateTime.MinValue;
var maxDateTime = DateTime.MaxValue;
```

After running the above example, `generatedDateTime` will be `1/1/0001 12:00:00 AM`, `minDateTime` will be (assuming it was run on Dec 2, 2013) `12/2/2013 12:00:00 AM`, and `maxDateTime` will be `12/31/9999 11:59:59 PM`. I'm fairly certain this is not intended.

**Comment by pjmagee** [comment]
Checkout my response on 34#issuecomment-137255280

**Comment by crmckenzie** [comment]
This issue has apparently been addressed by happenstance in version 5.0.0.

The following test has been added to the suite:

```csharp
        [Fact]
        public void DoesNotAlterStaticProperties() 
        {
            var minDateTime = DateTime.MinValue;
            var maxDateTime = DateTime.MaxValue;

            var generatedDateTime = Builder<DateTime>.CreateNew().Build();

            DateTime.MinValue.ShouldBe(minDateTime);
            DateTime.MaxValue.ShouldBe(maxDateTime);
        }
```
